src/main/java/org/llorllale/cactoos/matchers/Throws.java

```diff
@@ -88,33 +114,18 @@ protected boolean matchesSafely(final Scalar<T> obj,
             matches = false;
             dsc.appendText("The exception wasn't thrown.");
         } catch (final Exception cause) {
-            describe(dsc, cause.getClass(), cause.getMessage());
+            dsc
+                .appendText("Exception has type ")
+                .appendValue(cause.getClass())
```

Comment:
@victornoel Consider changing this to `cause.getClass().getName()` to keep this part of description behaviour unchanged.

Reply:
@vzurauskas good point, I changed too much things at once